content/user-authentication-in-python/index.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Implementing User Authentication in a Python Application
+date: "2021-06-29"
+coverImage: "coverImage.jpg"
+author: "Solomon Esenyi"
+tags: ["Python", "Authentication", "Flask"]
+---
```

Comment:
Add 1-2 liner description for your blog

For example
description: ""

Reply:
Thank you, I've made the necessary changes